src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -58,7 +57,6 @@ interface ExportAllOption {
 
 export interface SavedObjectsTableProps {
   allowedTypes: string[];
-  serviceRegistry: ISavedObjectsManagementServiceRegistry;
```

Comment:
Removes dependency on `serviceRegistry` from the top-most UI layer code.
We can handle the remaining cleanup in https://github.com/elastic/kibana/issues/67607.